virt-v2v/cold/entrypoint.go

```diff
@@ -74,6 +81,41 @@ func main() {
 	}
 }
 
+func getVmDiskPaths(domain *OvaVmconfig) []string {
+	var resp []string
+	for _, disk := range domain.Devices.Disks {
+		if disk.Source.File != "" {
+			resp = append(resp, disk.Source.File)
+		}
+	}
+	return resp
+}
+
+func customizeVM(source string, xmlFilePath string) error {
+	domain, err := GetDomainFromXml(xmlFilePath)
+	disks := getVmDiskPaths(domain)
+	if err != nil {
```

Comment:
the error should be checked before calling `getVmDiskPaths`, otherwise we may not get here..